packages/scripts/README.md

```diff
@@ -165,11 +189,15 @@ This is how you execute the script with presented setup:
 
 * `npm start` - starts the build for development.
 
+#### Advanced information
+
+It uses [webpack](https://webpack.js.org/) behind the scenes. It'll lookup for a webpack config in the top-level directory of your package and will use it if it finds one. If none is found, it'll use the default config bundled within `@wordpress/scripts` packages. Learn more in the [Advanced Usage](#advanced-usage) section.
+
 ### `test-e2e`
 
-Launches the End-To-End (E2E) test runner. It uses [Jest](https://jestjs.io/) behind the scenes and you are able to utilize all of its [CLI options](https://jestjs.io/docs/en/cli.html). You can also run `./node_modules/.bin/wp-scripts test:e2e --help` or `npm run test:e2e:help` (as presented below) to view all of the available options.
+Launches the End-To-End (E2E) test runner. Writing tests can be done using [Jest API](https://jestjs.io/docs/en/api) in combination with [Puppeteer API](https://github.com/GoogleChrome/puppeteer/blob/master/docs/api.md):
```

Comment:
```suggestion
Launches the End-To-End (E2E) test runner. Writing tests can be done using [Jest's API](https://jestjs.io/docs/en/api) in combination with [Puppeteer's API](https://github.com/GoogleChrome/puppeteer/blob/master/docs/api.md):
```

(Could've also suggested _"the JEST API"_, _"the Puppeteer API"_.)

Reply:
I will go with `the` form.